monero/backends/jsonrpc.py

```diff
@@ -183,6 +186,91 @@ def balances(self, account=0):
         _balance = self.raw_request('getbalance', {'account_index': account})
         return (from_atomic(_balance['balance']), from_atomic(_balance['unlocked_balance']))
 
+    def get_unspent_inputs(
+        self,
+        account_index: int = 0,
+        subaddr_indices: Optional[List[int]] = None,
+        verbose: bool = False
+    ) -> List[Input]:
+        """
+        Fetches the unspent inputs in the Wallet.
+
+        Args:
+            account_index: The index of the account whose Unspent Inputs we
+            want to fetch.
+            subaddr_indices: If it's provided, this function only fetches the
+            Unspent Inputs addressed to the Subaddress whose indices are
+            included in this list.
+            verbose: Whether to include key_image in the RPC response.
+        """
+        params = {
+            "transfer_type": "available",
+            'account_index': account_index,
```

Comment:
I tried to find what is common practice for strings `'` or `"`. Did not find anything so you choose but try to be consistent in one dict ;)

Reply:
Fixed. Standardised to `'`